core/src/replay_stage.rs

```diff
@@ -893,6 +902,9 @@ impl ReplayStage {
                         retransmit_not_propagated_time.as_us(),
                     );
                 }
+
+                drop(handle);
```

Comment:
why drop handle?

Reply:
senders are cloned when you create a handle. the bank_tx_execution.join() will drop bank_tx_execution.sender and once all the senders are dropped, bank_tx_execution will shutdown